src/App.jsx

```diff
@@ -39,6 +40,8 @@ class App extends Component {
             <Route path="/articles" component={AllArticles} />
             <PrivateRoute path="/profile" component={ProfilePage} exact />
             <PrivateRoute path="/profile/articles" component={AllUserArticles} exact />
+            <Route path="/profile" exact component={ProfilePage} />
```

Comment:
Why this again. It has already been defined as a `PrivateRoute` on line 41

Reply:
Thanks for spotting that. Will fix it